### DISCUSSION TITLE
Documentation for colonequals

### DISCUSSION THREAD

**Discussion by ronkok** [KaTeX/KaTeX#2650] [General] [answered]
KaTeX has several functions of the colonequals type. The ones that derive from the `mathtools` package are [documented](https://katex.org/docs/supported.html#relations). The ones that derive from `colonequals.sty` are not in the KaTeX documentation.

In the source code, these functions start at line 826 of `macros.js`

@edemaine, you created these functions. Is there any reason to omit them from the docs? If not, I'll work up a PR.

**Comment by edemaine** [comment, marked as answer]
No, that sounds like an oversight.  Thanks for spotting and offering to fix!  (and sorry for presumably forgetting to document them in the first place)
